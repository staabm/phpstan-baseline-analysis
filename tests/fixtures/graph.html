
            <!DOCTYPE HTML>
            <html>
            <head></head>
            <body>
                <script src="https://cdn.jsdelivr.net/npm/chart.js@3.9.1/dist/chart.min.js"></script>
                
                    <canvas id="chartContainerb56ee3eb73cd65912b11edba59cc68c2" style="height: 370px; width: 100%; margin-bottom: 30px;"></canvas>
                    <script>
                    (function () {
                        const ctx = document.getElementById('chartContainerb56ee3eb73cd65912b11edba59cc68c2');
                        const chart = new Chart(ctx, {
                            type: 'line',
                            options: {
                                plugins: {
                                    title: {
                                        display: true,
                                        text: 'PHPStan Baseline Analysis all-in.neon',
                                    }
                                }
                            },
                            data: {
                                labels:  [new Date(1664621068 * 1000).toLocaleDateString("de-DE")],
                                datasets: [{"label":"Overall-Errors","borderColor":"blue","data":[35]},{"label":"Classes-Cognitive-Complexity","borderColor":"#C24642","data":[70]},{"label":"Deprecations","borderColor":"lightgreen","data":[2]},{"label":"Invalid-Phpdocs","borderColor":"lightblue","data":[5]},{"label":"Unknown-Types","borderColor":"purple","data":[1]},{"label":"Anonymous-Variables","borderColor":"pink","data":[4]},{"label":"Native-Property-Type-Coverage","borderColor":"lightcoral","data":[1]},{"label":"Native-Param-Type-Coverage","borderColor":"lightseagreen","data":[27]},{"label":"Native-Return-Type-Coverage","borderColor":"lightsteelblue","data":[4]}]
                            }
                        });
                    })();
                </script>

                    <canvas id="chartContainer7b39166f92c8f32388d25c18147b8f5d" style="height: 370px; width: 100%; margin-bottom: 30px;"></canvas>
                    <script>
                    (function () {
                        const ctx = document.getElementById('chartContainer7b39166f92c8f32388d25c18147b8f5d');
                        const chart = new Chart(ctx, {
                            type: 'line',
                            options: {
                                plugins: {
                                    title: {
                                        display: true,
                                        text: 'PHPStan Baseline Analysis anonymous-variables.neon',
                                    }
                                }
                            },
                            data: {
                                labels:  [new Date(1664620967 * 1000).toLocaleDateString("de-DE")],
                                datasets: [{"label":"Overall-Errors","borderColor":"blue","data":[4]},{"label":"Classes-Cognitive-Complexity","borderColor":"#C24642","data":[0]},{"label":"Deprecations","borderColor":"lightgreen","data":[0]},{"label":"Invalid-Phpdocs","borderColor":"lightblue","data":[0]},{"label":"Unknown-Types","borderColor":"purple","data":[0]},{"label":"Anonymous-Variables","borderColor":"pink","data":[4]},{"label":"Native-Property-Type-Coverage","borderColor":"lightcoral","data":[0]},{"label":"Native-Param-Type-Coverage","borderColor":"lightseagreen","data":[0]},{"label":"Native-Return-Type-Coverage","borderColor":"lightsteelblue","data":[0]}]
                            }
                        });
                    })();
                </script>

            </body>
            </html>
        