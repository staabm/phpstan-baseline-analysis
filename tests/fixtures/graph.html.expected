
            <!DOCTYPE HTML>
            <html>
            <head></head>
            <body>
                <script src="https://cdn.jsdelivr.net/npm/chart.js@3.9.1/dist/chart.min.js"></script>
                
                    <canvas id="chartContainerb56ee3eb73cd65912b11edba59cc68c2" style="height: 370px; width: 100%; margin-bottom: 30px;"></canvas>
                    <script>
                    (function () {
                        const ctx = document.getElementById('chartContainerb56ee3eb73cd65912b11edba59cc68c2');
                        const chart = new Chart(ctx, {
                            type: 'line',
                            options: {  
                                plugins: {
                                    title: {
                                        display: true,
                                        text: 'PHPStan Baseline Analysis all-in.neon',
                                    }
                                },
                                scales: {
                                    x: {
                                        beginAtZero: true,
                                    },
                                    y: {
                                        beginAtZero: true,
                                    },
                                    yPercent: {
                                        min: 0,
                                        max: 100,
                                        beginAtZero: true,
                                        position: 'right',
                                        grid: {
                                            display: false,
                                        },
                                    },
                                }
                            },
                            data: {
                                labels:  [new Date(1664621068 * 1000).toLocaleDateString("de-DE")],
                                datasets: [{"label":"Overall-Errors","borderColor":"blue","data":[35]},{"label":"Classes-Cognitive-Complexity","borderColor":"#C24642","data":[70]},{"label":"Deprecations","borderColor":"lightgreen","data":[2]},{"label":"Invalid-Phpdocs","borderColor":"lightblue","data":[5]},{"label":"Unknown-Types","borderColor":"purple","data":[1]},{"label":"Anonymous-Variables","borderColor":"pink","data":[4]},{"label":"Native-Property-Type-Coverage","yAxisID":"yPercent","borderColor":"lightcoral","borderWidth":2,"type":"bar","data":[1]},{"label":"Native-Param-Type-Coverage","yAxisID":"yPercent","borderColor":"lightseagreen","borderWidth":2,"type":"bar","data":[27]},{"label":"Native-Return-Type-Coverage","yAxisID":"yPercent","borderColor":"lightsteelblue","borderWidth":2,"type":"bar","data":[4]},{"label":"Unused-Symbols","borderColor":"lightyellow","data":[]}]
                            }
                        });
                    })();
                </script>

                    <canvas id="chartContainer7b39166f92c8f32388d25c18147b8f5d" style="height: 370px; width: 100%; margin-bottom: 30px;"></canvas>
                    <script>
                    (function () {
                        const ctx = document.getElementById('chartContainer7b39166f92c8f32388d25c18147b8f5d');
                        const chart = new Chart(ctx, {
                            type: 'line',
                            options: {  
                                plugins: {
                                    title: {
                                        display: true,
                                        text: 'PHPStan Baseline Analysis anonymous-variables.neon',
                                    }
                                },
                                scales: {
                                    x: {
                                        beginAtZero: true,
                                    },
                                    y: {
                                        beginAtZero: true,
                                    },
                                    yPercent: {
                                        min: 0,
                                        max: 100,
                                        beginAtZero: true,
                                        position: 'right',
                                        grid: {
                                            display: false,
                                        },
                                    },
                                }
                            },
                            data: {
                                labels:  [new Date(1664620967 * 1000).toLocaleDateString("de-DE")],
                                datasets: [{"label":"Overall-Errors","borderColor":"blue","data":[4]},{"label":"Classes-Cognitive-Complexity","borderColor":"#C24642","data":[0]},{"label":"Deprecations","borderColor":"lightgreen","data":[0]},{"label":"Invalid-Phpdocs","borderColor":"lightblue","data":[0]},{"label":"Unknown-Types","borderColor":"purple","data":[0]},{"label":"Anonymous-Variables","borderColor":"pink","data":[4]},{"label":"Native-Property-Type-Coverage","yAxisID":"yPercent","borderColor":"lightcoral","borderWidth":2,"type":"bar","data":[0]},{"label":"Native-Param-Type-Coverage","yAxisID":"yPercent","borderColor":"lightseagreen","borderWidth":2,"type":"bar","data":[0]},{"label":"Native-Return-Type-Coverage","yAxisID":"yPercent","borderColor":"lightsteelblue","borderWidth":2,"type":"bar","data":[0]},{"label":"Unused-Symbols","borderColor":"lightyellow","data":[]}]
                            }
                        });
                    })();
                </script>

            </body>
            </html>
        